Items/de_DE

```diff
@@ -0,0 +1,87 @@
+misc.opened=Geöffnet
+misc.unopened=Ungeöffnet
+misc.north=Norden
+misc.south=Süden
+misc.east=Osten
+misc.west=Westen
+misc.lower=Unterer
+misc.upper=Oberer
+misc.leftHinge=Türangel links
+misc.rightHinge=Türangel rechts
+misc.powered=Aktiviert
+misc.unpowered=Deaktiviert
+misc.stage=Phase
+misc.flowing=Fließend
+misc.level=Höhe
+misc.source=Quellblock
+misc.still=Still
+misc.upright=Aufrecht
+misc.barkOnly=Nur Rinde
+misc.decayAfterTreeUpdate=Zerfallen nach Baum-Update
+misc.noDecay=Kein Zerfall
+misc.decay=Zerfall
+misc.decaying=Zerfallen
+misc.unused=Ungenutzt
+misc.off=Aus
+misc.down=Runter
+misc.up=Hoch
+misc.on=An
+misc.foot=Fuß
+misc.occupied=Besetzt
+misc.ascending=Aufsteigend
+misc.extended=Erweitert
+misc.drops=Drops
+misc.pistonHead=Kolbenkopf
+misc.sticky=Klebrig
+misc.pistonExtension=Verschobener Block
+misc.block=Block
+misc.seamed=Kontur
+misc.seamless=Glatt
+misc.oldWood=Altes Holz
+misc.bottom=Unten
+misc.top=Oben
+misc.primedWhenPunched=Zündet beim Abbau
+misc.age=Alter
+misc.dry=Trocken
+misc.moisture=Feuchtigkeit
+misc.wet=Nass
+misc.southWest=Südwesten
+misc.northWest=Nordwesten
+misc.northEast=Nordosten
+misc.southEast=Südosten
+misc.curved=Gekrümmt
+misc.hasRecord=Hat Schallplatte
+misc.bites=Bisse
+misc.delay=Verzögerung
+misc.allInside=Alle darin
+misc.center=Zentrieren
+misc.stem=Stem
+misc.allOutside=Alle außen
+misc.allStem=Alle Stem
```

Comment:
I did not know how to translate it so i kept it ^^ I don't really know what's meant by "stem" in this context. Do you have an idea?